src/static/js/ours/utils.js

```diff
@@ -62,6 +62,31 @@ var pretty_date = function (date_string) {
     }
 }
 
+var deadline_date = function (date_string) {
```

Comment:
Does this work well? Do we already have a time library *clientside*? `luxon` I think? Django already has functions to do this kinda stuff, as well. We could send back `start` of competition as well as `start_readable`?

Really want to stay away from any custom DateTime functions if we can help it. They get reaaaaaaaaaaaaaal finnicky

Reply:
Huh, looks like `moment` supports it but `luxon` doesn't?? https://github.com/moment/luxon/blob/master/docs/moment.md#humanization

We want to avoid doing this, if possible, but if we have to do it like this that's not the end of the world

Reply:
I was following along with the pretty_date function that was used earlier. Let me know if we want to change this.